addons/web_editor/static/lib/odoo-editor/test/spec/color.test.js

```diff
@@ -39,4 +39,22 @@ describe('applyColor', () => {
             contentAfter: '<p>ab<font style="background-color: rgb(255, 0, 0);">[]\u200B</font>cd</p>',
         });
     });
+    it('should apply a color on empty selection', async () => {
+        await testEditor(BasicEditor, {
+            contentBefore: '<p>[<br></p><p><br></p><p>]<br></p>',
+            stepFunction: setColor('rgb(255, 0, 0)', 'color'),
+            contentAfter: '<p><font style="color: rgb(255, 0, 0);">[<br></font></p>' +
+                          '<p><font style="color: rgb(255, 0, 0);"><br></font></p>' +
+                          '<p>]<br></p>',
```

Comment:
I think the end result in this case should be:
```html
<p><font style="color: rgb(255, 0, 0); " oe-zws-empty-inline>[\u200B</font></p>
<p><font style="color: rgb(255, 0, 0);" oe-zws-empty-inline>\u200B</font></p>
<p><font style="color: rgb(255, 0, 0);" oe-zws-empty-inline\u200B</font>]</p>
```
@Zinston What do you think ? Note that I have styled the third P whereas the suggested changes in this PR didn't. I would argue that if I selected three lines of my document and pressed a format button, it is because I want those three lines to be formatted that way. If there was content in all of those lines, it would be obvious. I think this might be a case where we specifically do _not_ want to correct a triple click selection.

Reply:
tldr;
I agree.

-----
That makes sense to me because it can't possibly be the result of a triple click since it's `<p>[...</p><p>...</p><p>]...</p>` and not just `<p>[...</p><p>]...</p>`. That middle `<p>` has to have been selected in a different way. So I'd say you're right unless the browser makes it unclear that line is selected (like there's no visual difference between when it is or isn't), but then again we might want to do it anyway if that's the only way to apply the color to every paragraph.